Spanner/tests/System/SpannerPgTestCase.php

```diff
@@ -120,8 +120,6 @@ public static function getDatabaseWithSessionPool($dbName, $options = [])
 
     public static function getDatabaseInstance($dbName)
     {
-        $keyFilePath = getenv('GOOGLE_CLOUD_PHP_TESTS_KEY_PATH');
```

Comment:
Is it removed by accident?

Reply:
No, it wasn't used in this function, so did a cleanup.